db/compaction/tiered_compaction_test.cc

```diff
@@ -215,8 +215,8 @@ TEST_F(TieredCompactionTest, SequenceBasedTieredStorageUniversal) {
   }
   ASSERT_OK(dbfull()->TEST_WaitForCompact());
 
-  // the penultimate level file temperature is not cold, all data are output to
-  // the penultimate level.
+  // the proximal level file temperature is not cold, all data are output to
```

Comment:
In my mind, "proximal" makes sense relative to the primary/normal output level of a compaction: it's the level before it. It doesn't really make sense in absolute terms. The penultimate level is the proximal level to the last level. I think "penultimate" still makes sense for cases in this file that are referring specifically to the next-to-last level, instead of to compaction things outside this file.

Not a big deal